File: api-gateway/Models/ServiceModels/Location/Address.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace api_gateway.Models.ServiceModels.Location
{
    public class Address
    {
        public City City { get; set; }
        public string PostalCode { get; set; }
        public string Street { get; set; }
        public int Number { get; set; }
        public string Addition { get; set; }
    }
}

File: ui/src/classes/Address.ts
import City from "./City";

export default class Address {
    public city: City;
    public street: string;
    public number: number;
    public addition: string;
    public postalCode: string;


    constructor(city: City, street: string, postalCode: string, number: number, addition: string){
        this.city = city;
        this.street = street;
        this.postalCode = postalCode;
        this.number = number;
        this.addition = addition;
    }
}